bindsnet/environment/__init__.py

```diff
@@ -126,12 +126,22 @@ def step(self, a: int) -> Tuple[torch.Tensor, float, bool, Dict[Any, Any]]:
             self.update_history()
             self.update_index()
 
-        self.obs = self.encoder(self.obs)
+        raw_obs = self.obs
+
+        # The new standard is BxTxCxHxW. The gym environment doesn't
+        # follow exactly the same protocol.
+        self.obs = self.encoder(self.obs).unsqueeze(1)
+        if self.obs.dim() == 4:
+            self.obs = self.obs.unsqueeze(2)
 
         self.episode_step_count += 1
 
         # Return converted observations and other information.
-        return self.obs, self.reward, self.done, info
+        return {"obs": self.obs,
```

Comment:
Why put this in a dictionary? This breaks the Gym environment standard

Reply:
Okay, I read up on the Gym environment standard and am now adding extra useful debug info (such as the original gym observation) to the info dictionary.